tests/drivers/spi/spi_slave_master_loopback/src/spi.c

```diff
@@ -0,0 +1,418 @@
+/*
+ * Copyright (c) 2017 BayLibre, SAS
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#define SYS_LOG_LEVEL SYS_LOG_LEVEL_DEBUG
+#include <logging/sys_log.h>
+
+#include <zephyr.h>
+#include <misc/printk.h>
+#include <string.h>
+#include <stdio.h>
+
+#include <spi.h>
+
+#ifdef CONFIG_TEST_SPI_MASTER_CS_GPIO_ENABLE
+struct spi_cs_control spi_master_cs = {
+	.gpio_pin = 0,
```

Comment:
Thanks a lot for the update!

I see that the gpio_dev field is set based on CONFIG_TEST_MASTER_CS_GPIO_DRV_NAME below in master_cs_ctrl_gpio_config(), but I don't see a config option or code for setting this gpio_pin field. Am I missing it?

Reply:
I added a comment explaining that

Reply:
I added the missing pin config